.travis.yml

```diff
@@ -20,6 +20,12 @@ cache:
     - ~/.composer/cache/files
 
 before_script:
+  # Update JS packages
+  - curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
```

Comment:
NVM has recently been added to the build-vm. Are these steps necessary?
See https://github.com/OpenConext/Stepup-Build/commit/2df4654580086cc3a2e5e1da9f2ae6fcb4db78ea

Reply:
On the Travis build environment, only an old unsupported Node version is available. This way we can install a version of our on liking. As this is on Travis. I don't think this affects the stepup build-vm. Or am I missing the point?